src/media_drv_output_dri.h

```diff
@@ -30,7 +30,7 @@
 #define _MEDIA__DRIVER_OUT_DRI_H
 #include <stdbool.h>
 #include "media_drv_defines.h"
-#define LIBVA_X11_NAME "libva-x11.so.1"
+#define LIBVA_X11_NAME "libva-x11.so"
```

Comment:
This patch seems wrong. The SONAME in libva-2.0.0+ is libva-x11.so.2 and end-users system should not have the unversionned symlink